lib/mirage/factories/node-policy.ts

```diff
@@ -0,0 +1,7 @@
+import { Factory } from 'miragejs'
+
+export const nodePolicy = Factory.extend({
+  read: 'ALLOW',
+  update: 'ALLOW',
+  delete: 'ALLOW',
```

Comment:
⏳ **Sand** - In the future, we may need to allow our developers to choose if they want to allow the dev user to edit or not. Maybe we could couple those variables to a given environment variable.

Reply:
This is an awesome addition, but this might produces a lot of unused environment variables, for that, I think we should look at that when this becomes a necessity, what do you think?

Maybe we could also get another way to change it without restarting the application.